src/Pickles/Pickles.TestFrameworks.UnitTests/NUnit/NUnit3/WhenDeterminingTheSignatureOfAnNunit3ExampleRow.cs

```diff
@@ -48,5 +48,18 @@ public void ThenCanSuccessfullyMatch()
             var isMatch = signature.IsMatch("AddingSeveralNumbers(\"40\",\"50\",\"90\",System.String[])".ToLowerInvariant());
             Check.That(isMatch).IsTrue();
         }
-    }
+
+		[Test]
+		public void ThenCanSuccesfullyMatchSpecialCharacters()
```

Comment:
"Successfully" is written with a double ss before the f :-)

Also please replace the spaces with tabs.